scripts/protoc-swagger-gen.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+proto_dirs=$(find ./proto -path -prune -o -name '*.proto' -print0 | xargs -0 -n1 dirname | sort | uniq)
+for dir in $proto_dirs; do
+
+  # generate swagger files (filter query files)
+  query_file=$(find "${dir}" -maxdepth 1 -name 'query.proto')
+  if [[ ! -z "$query_file" ]]; then
+    protoc  \
+    -I "proto" \
+    -I "third_party/proto" \
+    "$query_file" \
+    --swagger_out=logtostderr=true,stderrthreshold=1000,fqn_for_swagger_name=true,simple_operation_ids=true:.
+  fi
+done
+
+# combine swagger files
+# uses nodejs package `swagger-combine`.
+swagger-combine ./client/grpc-gateway/config.json -o ./client/grpc-gateway/swagger.json --continueOnConflictingPaths true --includeDefinitions true
```

Comment:
so for a custom SDK app you would need to add your own custom config.json? can we add a comment about this here?

Reply:
Other custom apps can also just use this `config.json`, they would need to add their swagger json urls in the `config.json`. I will add a comment